protostar/testing/cairo_cheatcodes/invoke_cairo_cheatcode.py

```diff
@@ -0,0 +1,28 @@
+import asyncio
+from typing import Any, Optional
+
+from protostar.starknet import Address, RawAddress, CairoOrPythonData
+from protostar.testing.cairo_cheatcodes.cairo_cheatcode import CairoCheatcode
+
+
+class InvokeCairoCheatcode(CairoCheatcode):
+    @property
+    def name(self) -> str:
+        return "invoke"
+
+    def build(self) -> Any:
+        return self.invoke
+
+    def invoke(
+        self,
+        contract_address: RawAddress,
+        function_name: str,
+        calldata: Optional[CairoOrPythonData] = None,
+    ):
+        asyncio.run(
+            self.cheaters.contracts.invoke(
+                function_name=function_name,
+                calldata=calldata,
+                contract_address=Address.from_user_input(contract_address),
+            )
+        )
```

Comment:
Nit: For other cheat codes we add a separate private method where we catch `ContractsCheaterException` and rethrow it as `CheatcodeException`. I think we should be consistent with this as long as you're not 100% sure that there's no way here for `ContractsCheaterException` to occur.